src/sentry/api/endpoints/internal/feature_flags.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+
+from django.conf import settings
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+import sentry
+from sentry.api.base import Endpoint, all_silo_endpoint
+from sentry.api.permissions import SuperuserPermission
+from sentry.app import env
+from sentry.conf.server import SENTRY_FEATURES_DESCRIPTIONS
+from sentry.runner.settings import configure, discover_configs
+
+
+@all_silo_endpoint
+class InternalFeatureFlagsEndpoint(Endpoint):
+    permission_classes = (SuperuserPermission,)
+
+    def get(self, request: Request) -> Response:
+        result = {}
+        print(settings.SENTRY_FEATURES)
+        for key in SENTRY_FEATURES_DESCRIPTIONS:
+            result[key] = {
+                "value": settings.SENTRY_FEATURES.get(key, False),
+                "description": SENTRY_FEATURES_DESCRIPTIONS[key],
+            }
+
+        return Response(result)
+
+    def put(self, request: Request) -> Response:
+        data = request.data.keys()
+        valid_feature_flags = [
+            flag for flag in data if SENTRY_FEATURES_DESCRIPTIONS.get(flag, False)
+        ]
+        _, py, yml = discover_configs()
+        # Open the file for reading and writing
+        with open(py, "r+") as file:
```

Comment:
I like this, the `sentry.conf.py` will be written to and updated automatically. It might be useful to also add a comment about the description of what a feature flag does? I think the `sentry.conf.example.py` in self-hosted should be up to date with the flags that get added to `SENTRY_FEATURES_DESCRIPTIONS` somehow too 🤔

Reply:
Fair enough, makes sense to me